SPDLOG_QUALITY_DECLARATION.md

```diff
@@ -0,0 +1,90 @@
+This document is a declaration of software quality for the `spdlog` external dependency imported by the `spdlog_vendor` package, based on the guidelines in [REP-2004](https://www.ros.org/reps/rep-2004.html).
+
+# `spdlog` Quality Declaration
+
+TODO: Declare the quality level for `spdlog`
+
+## Version Policy
+
+### Version Scheme
+
+There is a three-part version attached to `spdlog` releases.
+The sources define the version parts as `MAJOR`.`MINOR`.`PATCH`.
+There is an old ticket where the project's primary developer discusses versioning, but there is no documentation officially stating what approach the developer is taking.
+
+### API/ABI Stability Within a Released ROS Distribution
+
+As an external package, `spdlog` is not released on the same cadance as ROS.
+However, the `spdlog_vendor` package controls what version of `spdlog` is acceptable, so API/ABI stability within a ROS release can be enforced.
+
+### Public API Declaration
+
+Because `spdlog` can be consumed as a header-only library, nearly all of the source code can be considered part of the public API.
```

Comment:
It has a details namespace, is that the only part that should not be considered public?

Reply:
Given that there are `SPDLOG_API` declarations in headers within `details`, I would consider that to be part of the public API.